tests/cursor/bug1151-001.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+PHPC-1151: Segfault if session unset before first getMore (find)
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS_CRYPTO(); ?>
+<?php NEEDS('STANDALONE'); NEEDS_ATLEAST_MONGODB_VERSION(STANDALONE, "3.6"); ?>
```

Comment:
Previous [build failure](https://travis-ci.org/mongodb/mongo-php-driver/builds/365720995) was due to Travis attempting to run these tests on a pre-3.6 server. This adds the necessary skips from other session tests; however, it means that these tests won't run on Travis at all.

This would probably be a good time to bring over the [manual server installation we're currently doing in PHPLIB](https://github.com/mongodb/mongo-php-library/blob/master/.travis.yml).

Reply:
[PHPC-1159](https://jira.mongodb.org/browse/PHPC-1159).